libpod/rootless_cni_linux.go

```diff
@@ -0,0 +1,301 @@
+// +build linux
+
+package libpod
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+
+	cnitypes "github.com/containernetworking/cni/pkg/types/current"
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/containers/podman/v2/libpod/define"
+	"github.com/containers/podman/v2/libpod/image"
+	"github.com/containers/podman/v2/pkg/util"
+	"github.com/containers/storage/pkg/lockfile"
+	"github.com/hashicorp/go-multierror"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var rootlessCNIInfraImage = map[string]string{
+	// Built from https://github.com/AkihiroSuda/podman-network-create-for-rootless-podman .
+	// TODO: move to Podman's official quay
+	"amd64": "docker.io/akihirosuda/rootless-podman-network-sandbox:g56bd3cb-amd64",
```

Comment:
could we add the container image to the git repository?

Reply:
You mean ghcr.io ?

Reply:
I'd like a copy of the Dockerfile used to make it added somewhere in `contrib/` so we can regenerate if necessary.

Reply:
Moved to contrib